src/sentry/grouping/api.py

```diff
@@ -162,30 +223,28 @@ def get_default_enhancements(config_id=None) -> str:
     return Enhancements(rules=[], bases=[base]).dumps()
 
 
-def get_projects_default_fingerprinting_bases(
-    project: Project, config_id: str | None = None
-) -> Sequence[str] | None:
-    """Returns the default built-in fingerprinting bases (i.e. sets of rules) for a project."""
-    from sentry.projectoptions.defaults import DEFAULT_GROUPING_CONFIG
+def get_default_fingerprinting(config_id: str | None = None) -> dict[Any, Any]:
+    """Returns the default fingerprinting rules."""
+    bases: Sequence[str] | None = DEFAULT_GROUPING_FINGERPRINTING_BASES
 
-    config_id = (
-        config_id
-        # TODO: add fingerprinting config to GroupingConfigLoader and use that here
-        or PrimaryGroupingConfigLoader()._get_config_id(project)
-        or DEFAULT_GROUPING_CONFIG
-    )
+    if config_id is not None:
+        bases = CONFIGURATIONS[config_id].fingerprinting_bases
```

Comment:
Is there ever a chance of having an invalid config id here?

Reply:
We would have hit the problem in the previous code. I would not worry unless we're also changing the calling code.